package.json

```diff
@@ -44,7 +44,7 @@
   },
   "packageManager": "yarn@3.1.1",
   "resolutions": {
-    "@nuxtjs/i18n": "workspace:./packages/nuxt-i18n"
+    "@nuxtjs/i18n": "npm:@nuxtjs/i18n-edge@latest"
```

Comment:
Is this change really supposed to be committed?
I'm guessing that it would make local testing not use the local version of the package which could be quite inconvenient.